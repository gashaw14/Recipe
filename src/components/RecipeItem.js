import React from 'react';
import {Link} from 'react-router-dom';


const RecipeItem =({id,menuTitle,checkbox})=>(
    <div>
    <button className='big-button'>
    <Link to={`/edit/${id}`}>
    <h3>{menuTitle}</h3>
    <p>{checkbox && 'It has Ingredien'}</p>
    </Link>
    </button>
   
    </div>
)

export default RecipeItem;