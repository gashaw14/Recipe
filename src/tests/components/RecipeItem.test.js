import React from 'react';
import {shallow} from 'enzyme';
import RecipeItem from '../../components/RecipeItem';
import toJSON from 'enzyme-to-json';
const recipes =[{
    id:'1',
    menuTitle:'VEGI', 
    steps:'ttt',
    ingredient:'aaa',
    checkbox:false

},{
   id:'2',
    menuTitle:'meat', 
    steps:'cook',
    ingredient:'yyyy',
    checkbox:false
}
]
test('test recipe item component',()=>{
    const wrapper = shallow(<RecipeItem {...recipes[1]}/>)
    expect(toJSON(wrapper)).toMatchSnapshot();
})