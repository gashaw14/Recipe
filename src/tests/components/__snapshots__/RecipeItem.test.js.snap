// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test recipe item component 1`] = `
<div>
  <button>
    <Link
      to="/edit/2"
    >
      <h3>
        meat
      </h3>
      <p />
    </Link>
  </button>
</div>
`;
